common-app-lib/src/cmr/common_app/services/search/params.clj

```diff
@@ -99,8 +99,7 @@
 (defmethod parameter->condition :default
   [_context concept-type param value options]
   (errors/internal-error!
-    (format "Could not find parameter handler for [%s] with concept-type [%s]"
-            param concept-type)))
+   (format "Could not find parameter handler for [%s] with concept-type [%s]" param concept-type)))
```

Comment:
Not sure if you want this change - is it over 100 chars?

Reply:
No, it is no.